cmd/goProbe/config/config.go

```diff
@@ -124,12 +124,13 @@ type QueryRateLimitConfig struct {
 
 // APIConfig stores goProbe's API configuration
 type APIConfig struct {
-	Addr           string               `json:"addr" yaml:"addr"`
-	Metrics        bool                 `json:"metrics" yaml:"metrics"`
-	Profiling      bool                 `json:"profiling" yaml:"profiling"`
-	Timeout        int                  `json:"request_timeout" yaml:"request_timeout"`
-	Keys           []string             `json:"keys" yaml:"keys"`
-	QueryRateLimit QueryRateLimitConfig `json:"query_rate_limit" yaml:"query_rate_limit"`
+	Addr                string               `json:"addr" yaml:"addr"`
+	Metrics             bool                 `json:"metrics" yaml:"metrics"`
+	DisableIfaceMetrics bool                 `json:"disable_iface_metrics" yaml:"disable_iface_metrics"`
```

Comment:
In order to not break the API I've added the new flag (which will probably be used rarely anyway) next to the "main" key. In an ideal world this would be nested I guess, but a) this will introduce a breaking change which we'd have to explicitly handle (because you can't just default properly) and b) for consistency's sake you'd then have to introduce new nested options for `profiling` etc as well. So, nah...